notebooks/official/custom/SDK_Custom_Container_Prediction.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>Add run in Colab</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>